action.yaml

```diff
@@ -25,18 +25,14 @@ inputs:
       https://bazel.build/reference/command-line-reference#startup-options for a complete list. If
       unspecified, no startup options are specified.
     required: false
+  bazel-path:
+    description: A path to the Bazel executable. Defaults to PATH.
+    required: false
```

Comment:
true?

Reply:
intentionally required false, we'll fallback to a sane default if unspecified. i think slightly less config for the callsites is preferable